torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -852,6 +854,91 @@ def call_function(
         )
 
 
+class SdpaHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def create_wrapped_node(self, tx, args, kwargs):
+        # See NOTE [HigherOrderOperator tracing design] for more details
+        checkpoint = tx.copy_graphstate()
+        graph_checkpoint = tx.output.graph
+
+        example_vals = [torch.zeros((), dtype=args[0].dtype)] + [
+            torch.zeros((), dtype=torch.int) for _ in range(4)
+        ]
```

Comment:
Are they guaranteed to be contiguous?
Also, is FakeTensorMode active right now? (If not we should probably construct FakeTensors?)

Reply:
They're all scalar tensors. From my understanding faketensormode gets enabled separately right?